lena/homework_5/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pylint==2.6.0
+pytest==6.1.2
+pytest-cov==2.10.1
+black==20.8b0
```

Comment:
This tools only for development. It's better to put them in requirements.dev.txt.
And sort them in alphabetic order